repos/fdbt-site/src/pages/multiTapsPricing.tsx

```diff
@@ -0,0 +1,106 @@
+import React, { ReactElement, useState } from 'react';
+import { FullColumnLayout } from '../layout/Layout';
+import { MULTI_TAPS_PRICING_ATTRIBUTE } from '../constants/attributes';
+import TapPricingRow from '../components/TapPricingRow';
+import { ErrorInfo, NextPageContextWithSession, MultiTap } from '../interfaces';
+import ErrorSummary from '../components/ErrorSummary';
+import CsrfForm from '../components/CsrfForm';
+import { isWithErrors } from '../interfaces/typeGuards';
+import { getSessionAttribute } from '../utils/sessions';
+import { getCsrfToken } from '../utils';
+
+const title = 'Taps Pricing - Create Fares Data Service';
+const description = 'Pricing by multiple taps for capped product of the Create Fares Data Service';
+
+interface MultiTapsPricingProps {
+    errors?: ErrorInfo[];
```

Comment:
```suggestion
    errors: ErrorInfo[];
```
Doesnt need to be optional because in the prop's we're always sending across an array.